examples/src/constants.ts

```diff
@@ -17,3 +17,124 @@ export const agentColors: string[] = [
+    networkedTrajectory?: boolean;
+}
+
+export const TRAJECTORY_OPTIONS: TrajectoryOptions[] = [
+    {
+        id: "test_live_mode",
+        name: "test_live_mode",
+        remoteClientSimulator: true,
+        networkedTrajectory: true,
+    },
+    { id: "actin012_3.h5", name: "actin012_3.h5", networkedTrajectory: true },
+    {
+        id: "listeria_rocketbugs_normal_fine_2_filtered.simularium",
+        name: "listeria",
+        networkedTrajectory: true,
+    },
+    {
+        id: "kinesin002_01.h5",
+        name: "kinesin002_01.h5",
+        networkedTrajectory: true,
+    },
+    {
+        id: "microtubules038_10.h5",
+        name: "microtubules038_10.h5",
+        networkedTrajectory: true,
+    },
+    {
+        id: "microtubules_v2_shrinking.h5",
+        name: "microtubules_v2_shrinking.h5",
+        networkedTrajectory: true,
+    },
+    { id: "aster.cmo", name: "aster.cmo", networkedTrajectory: true },
+    {
+        id: "microtubules30_1.h5",
+        name: "microtubules",
+        networkedTrajectory: true,
+    },
+    {
+        id: "endocytosis.simularium",
+        name: "Actin in Clathrin-mediated Endocytosis",
+        networkedTrajectory: true,
+    },
+    {
+        id: "pc4covid19.simularium",
+        name: "SARS-CoV-2 Dynamics in Human Lung Epithelium",
+        networkedTrajectory: true,
+    },
+    {
+        id: "nanoparticle_wrapping.simularium",
+        name: "Membrane Wrapping a Nanoparticle",
+        networkedTrajectory: true,
+    },
+    {
+        id: "smoldyn_min1.simularium",
+        name: "Spatiotemporal oscillations in the E. coli Min system",
+        networkedTrajectory: true,
+    },
+    {
+        id: "smoldyn_spine.simularium",
+        name: "Sequestration of CaMKII in dendritic spines",
+        networkedTrajectory: true,
+    },
+    {
+        id: "medyan_Chandrasekaran_2019_UNI_alphaA_0.1_MA_0.675.simularium",
+        name: "Actin Bundle Dynamics with α–Actinin and Myosin",
+        networkedTrajectory: true,
+    },
+    {
+        id: "medyan_Chandrasekaran_2019_UNI_alphaA_0.1_MA_0.0225.simularium",
+        name: "Actin Bundle Dynamics with α–Actinin and Myosin",
+        networkedTrajectory: true,
+    },
+    {
+        id: "springsalad_condensate_formation_Below_Ksp.simularium",
+        name: "Condensate Formation",
+        networkedTrajectory: true,
+    },
+    {
+        id: "springsalad_condensate_formation_At_Ksp.simularium",
+        name: "Condensate Formation",
+        networkedTrajectory: true,
+    },
+    {
+        id: "springsalad_condensate_formation_Above_Ksp.simularium",
+        name: "Condensate Formation",
+        networkedTrajectory: true,
+    },
+    {
+        id: "blood-plasma-1.0.simularium",
+        name: "Blood Plasma",
+        networkedTrajectory: true,
+    },
+    { id: "TEST_POINTS", name: "Point Simulation", clientSimulator: true },
+    {
+        id: "TEST_LIVEMODE_API",
+        name: "Point Simulation LIVE",
```

Comment:
It's my understnading that both TEST_POINTS and TEST_LIVEMODE_API  are live simulations, just one uses ClientSimulator and one uses Octopus. Assuming I'm correct about that, could we change the names to be like `Point Simulation Octopus` and `Point Simulation ClientSim` or something?